stonks/__init__.py

```diff
@@ -0,0 +1,64 @@
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from webdriver_manager.firefox import GeckoDriverManager
+from bs4 import BeautifulSoup
+import time
+import re
+from .types import StonksResult
+
+
+class Handler:
+    """
+    Insert Your FII code to get the information about it
+     INPUT
+     - Ex: HGLG11
+
+     OUTPUT
+       {price: '00,00', div_yield: '0,00%', equity_value: '00,00', name:'HGLG11' }
+    """
+    base_url = 'https://www.fundsexplorer.com.br/funds'
```

Comment:
Please move the URL to a configuration file. `.env`.
You can use the name of `STONKS_TARGET_URL`